src/linea/LineaProver.ts

```diff
@@ -143,13 +148,14 @@ export class LineaProver extends BlockProver {
     };
   }
   async fetchProofs(target: HexString, slots: bigint[] = []) {
-    const ps = [];
+    const ps: Promise<RPCLineaGetProof>[] = [];
     for (let i = 0; ; ) {
       ps.push(
         // 20240825: most cloud providers seem to reject batched getProof
         // since we aren't in control of provider construction (ie. batchMaxSize)
         // sendImmediate is a temporary hack to avoid this issue
-        sendImmediate<RPCLineaGetProof>(this.provider, 'linea_getProof', [
+        // 20241027: switch to RetryProvider
```

Comment:
typo? GatewayProvider

Reply:
it was RetryProvider before I renamed it as a general HTTP provider solution to fix all ethers-related issues.